### DISCUSSION TITLE
Are there any realtime adaptions of Sirius Web  ?

### DISCUSSION THREAD

**Discussion by lak-proddev** [eclipse-sirius/sirius-components#3112] [Q&A]
Are there any real-time adaptions of Sirius Web?  Not a sample application.

**Comment by pcdavid** [comment]
Not sure if you mean "real-time" as in modelers for specifying [real-time systems](https://en.wikipedia.org/wiki/Real-time_computing) (then the answer is no, not that I know of at least), or "real worl" as in non-toy modelers. In the second case, yes there are, though they are not completely mature yet: see for example Eclipse SysON: https://mbse-syson.org/

**Reply by lak-proddev** [reply]
Thank you very much for the quick reply.

**Comment by sbegaudeau** [comment]
There is also the Sirius Web based version of Papyrus: https://gitlab.eclipse.org/eclipse/papyrus/org.eclipse.papyrus-web
